GAINS: {Kp: 15.0, Ki: 0.5, Kd: 0.01}
SET_POINT: 36.0
PIN: 26
FREQUENCY: 200
MAX_DUTYCYCLE: 100
SLEEP_TIME: 0.01
OUTPUT_LIMIT: {LOWER: 0, UPPER: 100}